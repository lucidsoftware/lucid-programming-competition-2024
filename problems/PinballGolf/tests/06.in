version https://git-lfs.github.com/spec/v1
oid sha256:34da484830c65a8b7afd41f554c998fa26c30e713817cd7b034ef07161f485bd
size 6429845
