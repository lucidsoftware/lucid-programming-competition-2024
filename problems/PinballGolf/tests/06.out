version https://git-lfs.github.com/spec/v1
oid sha256:90999b9c82ad56b828615ec4192b062a6374e28ea696639cdca7523e643bff67
size 12
